src/commons/zimbra-standard-colors.ts

```diff
@@ -3,13 +3,13 @@
  *
  * SPDX-License-Identifier: AGPL-3.0-only
  */
-export type ZimbraColor = {
+export type ZimbraColorType = {
 	color: string;
 	background: string;
 	label?: string;
 };
 
```

Comment:
@zovomat @nubsthead  can we create a separate ticket for replacing all the instances of this ZIMBRA_STANDARD_COLORS with the one that is exported from SHELL ?

Reply:
It would be a great idea, try to ask in the channel about it